sql/wsrep_schema.h

```diff
@@ -22,7 +22,7 @@
 
 #include "mysqld.h"
 #include "wsrep_mysqld.h"
-
+#include "log_event.h" // Gtid_log_event
```

Comment:
This could include directive could be avoided. Put forward declaration of Gtid_log_event in wsrep_schema.h, and include log_event.h in wsrep_schema.cc (if not already there).

Reply:
Fixed in 6092704619b